### ISSUE TITLE
"Mailbox items available to add-ins section" not being up to date. 

### ISSUE THREAD

**Issue by vlgiugal** [OfficeDev/office-js-docs-pr#2031]
Under the "Mailbox items available to add-ins" section, there is this statement which is not partially correct - not applying to all Outlook / Office versions. 

_**Outlook does not activate add-ins if the current message item, in a compose or read form, is one of the following:

Protected by Information Rights Management (IRM) or encrypted in other ways for protection. A digitally signed message is an example since digital signing relies on one of these mechanisms.**_

Upon testing, we have confirmation that the add-ins are not available for Outlook 2016 which **complies to the statement.**

This image is being shown:
![image](https://user-images.githubusercontent.com/68544548/87934265-62855880-ca97-11ea-86de-fd0a636fa142.png)

They are though available and usable in Outlook / Office 365 Build 11929.20776 which **does not comply to the statement.** 

Would there be possible a revise / article update? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3b2e062e-1a7a-1861-bf20-c23ac64d0d37
* Version Independent ID: ae2d1395-2a87-1bda-00fb-1b30f8e27b66
* Content: [Outlook add-ins overview - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview#mailbox-items-available-to-add-ins)
* Content Source: [docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/outlook/outlook-add-ins-overview.md)
* Product: **outlook**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by ElizabethSamuel-MSFT** [comment]
@vlgiugal Thanks for asking about this. We'll take a look and update accordingly.

Thanks.

**Comment by jeremy-msft** [comment]
@vlgiugal, thanks for your feedback. Could you please re-confirm for us the build number and version where you are seeing this issue?

**Comment by vlgiugal** [comment]
Hi @jeremy-msft, we've tested on Office 365 Build 11929.20776 - https://docs.microsoft.com/en-us/officeupdates/semi-annual-enterprise-channel#version-1908-may-12 and **it worked here.**  This is in contrast with what the article states. 

We've also tested on Outlook 2016 random builds - but there we couldn't reproduce the behavior - **as confirmed in the article.**  Here was no issue, as it complied to the article.

**Comment by exextoc** [comment]
We tried build 11929 and we cannot repro so far. Could you share more information about the repro steps? (for example, whether the addin is pinned, whether this as in read or compose, message or calendar?). Screenshots or a video can also be helpful for us to get a repro.

**Comment by MartinKarak** [comment]
Outlook Add-im Send to OneNote works with signed and also not signed e-mails
![O365-sig01](https://user-images.githubusercontent.com/69466226/89786758-bf1ae700-db1c-11ea-9016-b76d187bdab3.JPG)
![O365-sig02](https://user-images.githubusercontent.com/69466226/89786762-c0e4aa80-db1c-11ea-822c-098fc2d1720e.JPG)
![O365-sig03](https://user-images.githubusercontent.com/69466226/89786768-c215d780-db1c-11ea-9a8c-56c91c28e938.JPG)
![O365-NoSig02](https://user-images.githubusercontent.com/69466226/89786789-c93ce580-db1c-11ea-87c3-9dd487b3e011.JPG)
![O365-NoSig01](https://user-images.githubusercontent.com/69466226/89786794-ca6e1280-db1c-11ea-98a8-d26bf0a3aca0.JPG)

**Comment by exextoc** [comment]
Thanks for providing the screenshots. I see you have insights add-in as well, do you see a similar behavior with Insights as well or is that different? Does this behavior happen for other addins as well or only for OneNote?

**Comment by MartinKarak** [comment]
Hi,
I see the same behaviour.

**Comment by exextoc** [comment]
Sorry, there had been some confusion here, that I verified with the rest of the team. For Outlook 2013, and Outlook 2016, both IRM Encrypted and Digitally Signed Messages were excluded from Outlook Web Add-ins. Due to customer feedback, we relaxed this restriction for the C2R Outlook 365 versions, so that only Digitally SIGNED messages are allowed. IRM Encrypted messages are still currently not allowed. (though this may change in the future) 

I believe based on that information, your initial post was correct, and we should revise our documentation. Does that sound correct to you as well?
